insights/parsers/db2licm.py

```diff
@@ -0,0 +1,100 @@
+"""
+IBM DB2 Sever details
+=====================
+
+Module for the processing of output from the ``db2licm -l`` command.
+Data is avaliable as rows of the output contained
+in one ``Record`` object for each line of output.
+"""
+from insights.core.plugins import parser
+from insights.core import CommandParser, LegacyItemAccess
+from insights.parsers import ParseException, get_active_lines
+from insights.specs import Specs
+
+
+@parser(Specs.db2licm_l)
+class DB2Info(LegacyItemAccess, CommandParser):
```

Comment:
The `LegacyItemAccess` will be deprecated, would you please to mix it with `dict` directly, which provides all the same access features as `LegacyItemAccess`.

See: https://github.com/RedHatInsights/insights-core/blob/b6da835477b70f7898985a7fb61a403152784983/insights/parsers/rndc_status.py#L11